integreat_cms/cms/forms/custom_content_model_form.py

```diff
@@ -98,7 +98,7 @@ def clean(self) -> dict[str, Any]:
 
         return cleaned_data
 
-    # pylint: disable=too-many-branches
```

Comment:
96 lines for one function, two disabled pylint rules and a function that does a lot of things are not really great in my opinion. Could you please open up an issue to refactor this function? Maybe for a milestone in the near future? :)

Reply:
Agree 😅 See #2849